### ISSUE TITLE
[Epic] Improve post-Getting Started experience: GA

### ISSUE THREAD

**Issue by davidwrede** [pulumi/docs#5010]
Currently, our getting started tutorials provide a "Next Steps" page with a list of links for users to deepen their knowledge of Pulumi. The links provided on these pages are manually curated and and may not accurately reflect the specific areas in which users want to learn more about and/or may not be centered around their particular use case(s).

To improve on the post-getting started experience, we should make the following improvements in Q1.

## Work items

**Cost in weeks:** 11

#### M52

- [x] Epic: Create (and direct people to) a more useful site search experience that helps people find the content that aligns with their needs (#4642)
- [x] Create tracking spreadsheet for language parity across top five tutorials for AWS, Azure, Google Cloud, and Kubernetes

#### M54

- [ ] To achieve language parity across top five tutorials, create missing code examples/tutorials for AWS (chosen based on quarterly page views) 
- [ ] Update the list of tutorials for the AWS "Next Steps" page to showcase the top five most viewed tutorials
- [ ] Make sure the top five AWS tutorials are listed in "Featured Tutorials" section
- [ ] Add section to "Next Steps" pages that lists (and links to) upcoming workshops
- [x] Add link to [Adopting Pulumi](https://www.pulumi.com/docs/guides/adopting/import/) section for users who want to import existing resources and manage them with Pulumi

#### M55

- [x] To achieve language parity across top five tutorials, create missing code examples/tutorials for Azure (chosen based on quarterly page views)
- [ ] Update the list of tutorials for the Azure "Next Steps" pages to showcase the top five most viewed tutorials
- [ ] Make sure the top five Azure tutorials are listed in "Featured Tutorials" section
- [ ] Create missing code examples/tutorials for Kubernetes
- [ ] Update the list of tutorials for the Kubernetes "Next Steps" page to showcase the top five most viewed tutorials
- [ ] Make sure the top five Kubernetes tutorials are listed in "Featured Tutorials" section

#### Nice-to-haves
- [ ] Improve Google Cloud tutorials experience
- [ ] Improve Outputs content leveraging internal draft content

### Launch readiness 🎁

**Release coordinator:** @davidwrede

- [ ] Demo the updates at TGIF
  - [x] Swiftype search
  - [ ] New tutorials
- [ ] Docs 
- [ ] Social media & community Slack 
  - [x] [Swiftype announcement](https://pulumi-community.slack.com/archives/CB36DSVSA/p1613784279000300)
  - [ ] New tutorials
- [ ] Ship it 🚀

## [Objective and Key Results](https://docs.google.com/document/d/1bg8kqBLUNhHxWACbzWAjeFng445a5JDYnZnvygC3E6c/edit#)

- **Objective:** Grow WRAU 17% from 940 to 1,100 (5.4% MoM).
  - **Key result:** Deliver 3 measurable improvements to AU funnel metrics (Reduce first-run error rate, post-getting-started content, TBD, etc.)

**Comment by infin8x** [comment]
#### Status March 12: 🟡 

- This work is making good progress but is behind schedule due to some unplanned personal time on the team

**Comment by davidwrede** [comment]
#### Status April 2: 🟡 

- Azure examples PR up for review. Tutorial content update PR scheduled for 4/4.
- AWS example updates in-flight. Example PRs and tutorial content update PR scheduled for 4/9.
- With remaining time, focus will be on Kubernetes to get in before PulumiUp. Google Cloud tutorials and examples will come after.

**Comment by davidwrede** [comment]
**Status April 28:** 🟡 

- Progress on tutorial content updates for Azure and AWS were halted a couple weeks back to work on Pulumi 3.0 deliverables. Work on these updates has resumed. PR for Azure tutorial updates scheduled for 5/06.
- Remaining AWS language-parity examples are almost done. Blocked on a few issues, but we have a meeting on 4/30 to review and resolve them.
- AWS tutorial updates scheduled for review on 5/13.
- Started on Kubernetes language-parity examples. Will shift more time to this after the Azure and AWS tutorial updates are in review.

**Comment by davidwrede** [comment]
Closing this epic in favor of opening and tracking individual tasks as issues to get more clarity on progress.

**Comment by davidwrede** [comment]
Individual work items are now tracked as separate issues in pulumi/home.
